package.json

```diff
@@ -34,32 +34,32 @@
   "license": "MIT",
   "repository": "jest-community/jest-extended",
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/eslint-parser": "^7.14.7",
-    "@babel/preset-env": "^7.14.7",
-    "@types/jest": "^24.0.0",
-    "babel-jest": "^24.0.0",
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.5",
+    "@babel/eslint-parser": "^7.15.7",
+    "@babel/preset-env": "^7.15.6",
+    "@types/jest": "^27.0.2",
+    "babel-jest": "^27.2.4",
     "babel-jest-assertions": "^0.1.0",
     "babel-plugin-gwt": "^1.0.0",
-    "eslint": "^7.30.0",
+    "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jest": "^24.3.6",
-    "eslint-plugin-prettier": "^3.4.0",
-    "husky": "^7.0.1",
-    "import-all.macro": "^2.0.3",
-    "jest": "^24.0.0",
-    "jest-each": "^0.5.0",
-    "jest-watch-typeahead": "^0.2.0",
-    "lint-staged": "^11.0.0",
-    "prettier": "^2.3.2",
-    "pretty-format": "^22.1.0"
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-jest": "^24.5.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "husky": "^7.0.2",
+    "import-all.macro": "^3.1.0",
+    "jest": "^27.2.4",
+    "jest-each": "^27.2.4",
+    "jest-watch-typeahead": "^1.0.0",
+    "lint-staged": "^11.2.0",
+    "prettier": "^2.4.1",
+    "pretty-format": "^27.2.4"
   },
   "dependencies": {
-    "expect": "^24.1.0",
-    "jest-get-type": "^22.4.3",
-    "jest-matcher-utils": "^22.0.0"
+    "expect": "^27.2.4",
+    "jest-get-type": "^27.0.6",
+    "jest-matcher-utils": "^27.2.4"
```

Comment:
only these are needed, please leave the other dependencies alone (to reduce the diff and keep the PR focused)